openfisca_us/variables/gov/states/mo/tax/income/adjusted_gross_income/mo_taxable_income.py

```diff
@@ -0,0 +1,42 @@
+from openfisca_us.model_api import *
+
+
+class mo_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO taxable income"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-A_2021.pdf"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        threshold = parameters(period).gov.states.mo.tax.income.mo_taxable_income.mo_federal_itemization_threshold[filing_status]
+
+        #any itemizations 
+        agi = tax_unit("adjusted_gross_income", period)
+        total_itemized_federal_deductions = tax_unit("mo_federal_itemized_deductions", period)
+        if total_itemized_federal_deductions > 0:
+            
+            person = tax_unit.members
+            social_security_tax = person("employee_social_security_tax", period)
+            medicare_tax = person("employee_medicare_tax", period)
+            self_employment_tax = person("self_employment_tax", period)
+            items = (total_itemized_federal_deductions + social_security_tax + medicare_tax + self_employment_tax)
```

Comment:
Instead of the above
```suggestion
            items = add(person, period, ["employee_social_security_tax", "employee_medicare_tax", "self_employment_tax"])
```